components/site-search/server.js

```diff
@@ -11,6 +11,7 @@ export class SiteSearch extends ServerComponent {
     return PageLayout.render(
       context,
       html`<div class="site-search">
+        <h1 class="visually-hidden">${context.l10n`Search`}</h1>
```

Comment:
Perhaps "Search Results"? Provides a bit more of a clue there's results after the input